fontforgeexe/math.c

```diff
@@ -1799,6 +1799,7 @@ static void MKDInit(MathKernDlg *mkd,SplineChar *sc) {
     mkd->dummy_sf.glyphs = mkd->chars;
     mkd->dummy_sf.glyphcnt = mkd->dummy_sf.glyphmax = 4;
     mkd->dummy_sf.pfminfo.fstype = -1;
+    mkd->dummy_sf.pfminfo.stylemap = -1;
```

Comment:
I don't esp. want to tackle this just now, but you are right that there is duplication with the other function.